package.json

```diff
@@ -16,6 +16,9 @@
     "make": "export NODE_ENV=production && node scripts/build.js && electron-forge make -p 'darwin' && electron-forge make -p 'win32'"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
```

Comment:
i'm really hoping to remove all carrots (^) before version numbers in the package.json.
from inspecting package-lock, it looks like you should lock them at these specific versions:
- babel/polyfill - 7.2.5
- @ledgerhq/hw-app-btc - 4.39.0
- @ledgerhq/hw-transport-node-hid - 4.40.0

The auto-updating from the (^) has been annoying to deal with, because often these updates introduce security warnings from npm audit that are a pain to deal with